### PR TITLE
fix: Use contentheaders for new versions of restsharp

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs [hunk 1]
```diff
@@ -186,11 +186,11 @@ private async Task FetchConfigAsyncWithTask()
             {
                 try
                 {
-                    var lastModified = res.Headers?.FirstOrDefault(e => e.Name?.ToLower() == "last-modified");
-                    var etag = res.Headers?.FirstOrDefault(e => e.Name?.ToLower() == "etag");
-                    if (!string.IsNullOrEmpty(configLastModified) && lastModified != null && !string.IsNullOrEmpty((string)lastModified.Value) )
+                    var lastModified = res.ContentHeaders?.FirstOrDefault(e => e.Name?.ToLower() == "last-modified")?.Value as string;
```
Comment (kaushalkapasi) [inline DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:189]: what will the default be here if there's no "first"?

Comment (JamieSinn) [reply DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:189]: null

Comment (kaushalkapasi) [reply DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:189]: Will that be handled correctly during reading it or do we need a default of empty string?

Comment (JamieSinn) [reply DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:189]: it's handled properly by the isnullorempty check below

**Review by kaushalkapasi — APPROVED** [review]

**Pull request merged**